### ISSUE TITLE
[Taskbar Labels for Windows 11] Does not work upon restart, using dev channel build

### ISSUE THREAD

**Issue by m417z** [ramensoftware/windhawk-mods#156]
> My Windows build is 25247.rs_prerelease.221114-1311 Dev Chanenel, I have restart my computer and Taskbar Labels continued to not run upon starting explorer.exe. 
> ![image](https://user-images.githubusercontent.com/71511385/204164221-561d76fa-f4f3-4007-9800-8f38140d3b84.png)
> ![image](https://user-images.githubusercontent.com/71511385/204164236-39dee8ee-fe45-4b74-9bcf-e72a302f8789.png)

_Moved from https://github.com/m417z/my-windhawk-mods, originally posted by @tobyisawesome on Nov 28, 2022_

**Comment by m417z** [comment]
@tobyisawesome Fixed in version 1.1.2.

**Comment by m417z** [comment]
Actually, the fix didn't make it to version 1.1.2 because of a small mistake. It will be included in the next version. For now, as a temporary solution, you can fork the mod and replace these lines:
```cpp
                LR"(public: virtual long __cdecl CTaskListWnd::TaskDestroyed(struct ITaskGroup *,struct ITaskItem *,enum TaskDestroyedFlags))",
                LR"(public: virtual long __cdecl CTaskListWnd::TaskDestroyed(struct ITaskGroup * __ptr64,struct ITaskItem * __ptr64,enum TaskDestroyedFlags) __ptr64)",
```
with this:
```cpp
                LR"(public: virtual long __cdecl CTaskListWnd::TaskDestroyed(struct ITaskGroup *,struct ITaskItem *))",
                LR"(public: virtual long __cdecl CTaskListWnd::TaskDestroyed(struct ITaskGroup * __ptr64,struct ITaskItem * __ptr64) __ptr64)",
```

**Comment by m417z** [comment]
The fix in now available in version 1.1.3. Let me know if there are other incompatibilities with preview versions.
